README.md

```diff
@@ -90,6 +90,12 @@ allprojects {
 
 ## API Reference
 
+#### setApiKey(): void
```

Comment:
You have forgotten `setApiKey(apiKey: string): void`, haven't it?

Reply:
Indeed! Fixed.